packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -194,7 +194,15 @@ const GeneralTab = decorate([
                 <Icon icon='memory' size='lg' />
               </span>
             </h2>
-            <BlockLink to={`/vms/${id}/stats`}>{statsOverview && <MemorySparkLines data={statsOverview} />}</BlockLink>
+            {vm.managementAgentDetected === true && vm.managementAgentDetected !== undefined ? (
```

Comment:
Is there a reason to have this very specific condition test? Or can it be replaced by this?
```suggestion
            {vm.managementAgentDetected ? (
```